addon/templates/components/bourbon-text-field.hbs

```diff
@@ -1,2 +1,2 @@
 <label class="BourbonTextField-label">{{placeholder}}</label>
-<input class="BourbonTextField-input" type="text" placeholder={{placeholder}} value={{value}}>
+<input class="BourbonTextField-input" type={{type}} placeholder={{placeholder}} value={{value}} disabled={{disabled}} autocomplete={{autocomplete}} autofocus={{autofocus}} readonly={{boundReadOnly}}>
```

Comment:
How do (did?) we use the `type` attribute for the `bourbon-text-field` (previously `text-field`?) component? And what happens if the consumer doesn't pass in a `type`?

Reply:
i think it i was going a bit crazy.  it was `type="text"` in the old `text-field` and probably should be that since it is an "text-field".    :)

Reply:
yeah, thought i'd ask, in case we needed to support text-like types, e.g. number or password or something, but i guess not haha